### PR TITLE
CORS for media API

### PR DESCRIPTION
**Pull request by markeloff86**

### CODE REVIEW

File: server/media/media.go [hunk 1]
```diff
@@ -97,7 +97,7 @@ func matchCORSMethod(allowMethods []string, method string) bool {
 	return false
 }
 
-// CORSHandler is the default CORS processor for use by media handlers.
+// CORSHandler is the default preflight OPTIONS processor for use by media handlers.
```
Comment (or-else) [inline server/media/media.go:100]: It's not just a preflight processor, it adds CORS headers to regular requests too.

File: server/media/s3/s3.go [hunk 2]
```diff
@@ -151,13 +151,10 @@ func (ah *awshandler) Init(jsconf string) error {
 	return err
 }
 
-// Headers redirects GET, HEAD requests to the AWS server.
+// Headers redirects to the AWS server.
```
Comment (or-else) [inline server/media/s3/s3.go:154]: That's not what the code does.

**Comment by or-else** [comment]
I'm going to merge your change and then make changes.

**Comment by markeloff86** [comment]
Oh, sorry!
All comments changes are from my old code :(
It was merge mistake.

**Pull request merged**